docs/README.md

```diff
@@ -16,6 +16,7 @@ This is the documentation of ActiveModelSerializers, it's focused on the **0.10.
 - [Instrumentation](general/instrumentation.md)
 - [JSON API Schema](jsonapi/schema.md)
 - [ARCHITECTURE](ARCHITECTURE.md)
+- [Passing Arbitrary Options](general/passing_arbitrary_options.md)
```

Comment:
wondering if this should be in `howto/`. I'm thinking `serializers` (instance_options)  and `rendering` (passing artbitrary optios) in `general/` should reference this file located in `howto/`.